src/write_outputs/write_vre_stor.jl

```diff
@@ -0,0 +1,429 @@
+received this license file.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
+@doc raw"""
+	write_vre_stor(path::AbstractString, inputs::Dict, setup::Dict, EP::Model)
+
+Function for writing the vre-storage specific files.
+"""
+
+function write_vre_stor(path::AbstractString, inputs::Dict, setup::Dict, EP::Model)
+
+	### CAPACITY DECISIONS ###
+	dfVreStor = write_vre_stor_capacity(path, inputs, setup, EP)
+
+	### CHARGING DECISIONS ###
+	write_vre_stor_charge(path, inputs, setup, EP)
+
+	### DISCHARGING DECISIONS ###
+	write_vre_stor_discharge(path, inputs, setup, EP)
+
+	return dfVreStor
+end
+
+@doc raw"""
+	write_vre_stor_capacity(path::AbstractString, inputs::Dict, setup::Dict, EP::Model)
+
+Function for writing the vre-storage capacities.
+"""
+function write_vre_stor_capacity(path::AbstractString, inputs::Dict, setup::Dict, EP::Model)
+	VRE_STOR = inputs["VRE_STOR"]
+	SOLAR = inputs["VS_SOLAR"]
+	WIND = inputs["VS_WIND"]
+	DC = inputs["VS_DC"]
+	STOR = inputs["VS_STOR"]
+	dfGen = inputs["dfGen"]
+	dfVRE_STOR = inputs["dfVRE_STOR"]
+
+	MultiStage = setup["MultiStage"]
+
+	# Solar capacity
+	capsolar = zeros(size(inputs["RESOURCES_VRE_STOR"]))
+	retcapsolar = zeros(size(inputs["RESOURCES_VRE_STOR"]))
+	existingcapsolar = zeros(size(inputs["RESOURCES_VRE_STOR"]))
+
+	# Wind capacity
+	capwind = zeros(size(inputs["RESOURCES_VRE_STOR"]))
+	retcapwind = zeros(size(inputs["RESOURCES_VRE_STOR"]))
+	existingcapwind = zeros(size(inputs["RESOURCES_VRE_STOR"]))
+
+	# Inverter capacity
+	capdc = zeros(size(inputs["RESOURCES_VRE_STOR"]))
+	retcapdc = zeros(size(inputs["RESOURCES_VRE_STOR"]))
+	existingcapdc = zeros(size(inputs["RESOURCES_VRE_STOR"]))
+
+	# Grid connection capacity
+	capgrid = zeros(size(inputs["RESOURCES_VRE_STOR"]))
+	retcapgrid = zeros(size(inputs["RESOURCES_VRE_STOR"]))
+	existingcapgrid = zeros(size(inputs["RESOURCES_VRE_STOR"]))
+
+	# Energy storage capacity
+	capenergy = zeros(size(inputs["RESOURCES_VRE_STOR"]))
+	retcapenergy = zeros(size(inputs["RESOURCES_VRE_STOR"]))
+	existingcapenergy = zeros(size(inputs["RESOURCES_VRE_STOR"]))
+
+	# Charge storage capacity DC
+	capchargedc = zeros(size(inputs["RESOURCES_VRE_STOR"]))
+	retcapchargedc = zeros(size(inputs["RESOURCES_VRE_STOR"]))
+	existingcapchargedc = zeros(size(inputs["RESOURCES_VRE_STOR"]))
+
+	# Charge storage capacity AC
+	capchargeac = zeros(size(inputs["RESOURCES_VRE_STOR"]))
+	retcapchargeac = zeros(size(inputs["RESOURCES_VRE_STOR"]))
+	existingcapchargeac = zeros(size(inputs["RESOURCES_VRE_STOR"]))
+
+	# Discharge storage capacity DC
+	capdischargedc = zeros(size(inputs["RESOURCES_VRE_STOR"]))
+	retcapdischargedc = zeros(size(inputs["RESOURCES_VRE_STOR"]))
+	existingcapdischargedc = zeros(size(inputs["RESOURCES_VRE_STOR"]))
+
+	# Discharge storage capacity AC
+	capdischargeac = zeros(size(inputs["RESOURCES_VRE_STOR"]))
+	retcapdischargeac = zeros(size(inputs["RESOURCES_VRE_STOR"]))
+	existingcapdischargeac = zeros(size(inputs["RESOURCES_VRE_STOR"]))
```

Comment:
make the interface to the `inputs` dictionary smaller: access `size(inputs["RESOURCES_VRE_STOR"])` once and set the result to a variable.